torchvision/prototype/transforms/__init__.py

```diff
@@ -1,4 +1,3 @@
-from . import functional
-from .functional import InterpolationMode  # usort: skip
-
+from . import kernels  # usort: skip
+from . import functional  # usort: skip
```

Comment:
We need to keep the order here since `functional` depends on `kernels`. Otherwise you cannot do `from torchvision.prototype.transforms import kernels as K` in `torchvision.prototype.transforms.functional`.